cli/types/cypress.d.ts

```diff
@@ -3013,7 +3013,7 @@ declare namespace Cypress {
     videoCompression: number | boolean
     /**
      * Whether Cypress will record a video of the test run when running headlessly.
```

Comment:
Should this be calling out cypress run mode? do we really only recorded when headless or just run mode? haven't seen this blurb in our docs anywhere if it is only headless mode.

Reply:
I think you're right. recording happens in run mode regardless of whether the browser is headed or not. I'll push an update to call out run mode specifically

Reply:
hows 4ef0364c0c look?